src/main/asciidoc/inc/external/_property_configuration.adoc

```diff
@@ -327,12 +327,26 @@ configuration (since version 0.25.0), which can have one of the following values
 | Effectively disable properties, same as not specifying the `<external>` section at all.
 |===
 
-For single-image configurations it is also possible to active property
-configuration by setting the property `docker.imagePropertyConfiguration` to a
-valid `property mode`, without adding a `<external>` section.
-This will use properties with default prefix.
-This can be useful if most of configuration is specified in XML/POM file, but there
-is need to override certain configuration values without altering the POM file.
+[[combining-property-config-externally]]
+.Activating property configuration externally
+It also possible to activate property configuration by setting the property `docker.imagePropertyConfiguration` to a
+valid `property mode`, without adding an `<external>` section. The plugin will then use any properties with default `docker.` prefix.
+This can be useful if most of the configuration is specified in XML/POM file, but there
+is need to override certain configuration values without altering the POM file (instead add this to a parent POM or
+global settings.xml).
+
+If set in parent POM, but not wanted in specific project, the property could be overriden locally with the value `skip`
+to disabled property configuration for that particular project.
+If set in settings.xml however, by Maven design, that value will always take precedence over any properties defined in
+pom.xml.
+
+For configurations with multiple images, using this property will by default produce an error. All images would then
+use the same `docker` property prefix, resulting in multiple identical configurations.
+This can be overruled by adding an explicit <external> configuration element with an explicit <prefix> to all
+images (or at least all but one). Normally you'd want to use different prefix for each image, but if explicitly set
+it does allow you to use the same prefix (even `docker`) on all images. This is useful in case you just want to share
+a few properties. This only makes sense when `property mode` is _override_ or _fallback_ and image-specific configuration
+are defined in the POM configuration.
```

Comment:
Can we add an example here please, e.g. the example for setting the http proxy for all configured images with a single property (i.e. you use case) ? I think this would help to understand people why it would make sense sometimes to use one property for all configured images.

Reply:
Good point. An example has been added for http_proxy and a single image, as well as an example with two images (using the same prefix).